src/mmrelay/meshtastic_utils.py

```
@@ -435,6 +569,138 @@ def _get_device_metadata(client: Any) -> dict[str, Any]:
+
+        # Always call close() to release resources
+        logger.debug(f"Closing BLE interface ({reason})")
+
+        # For BLE interfaces, explicitly disconnect the underlying BleakClient
+        # to prevent stale connections in BlueZ (official library bug)
+        if hasattr(iface, "client"):
+            logger.debug(f"Explicitly disconnecting BLE client ({reason})")
+            try:
+                iface.client.disconnect()
+            except Exception:
+                pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)